### DISCUSSION TITLE
2023년 11월 25일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#99] [General]
# 2023년 11월 25일

TODO LOG 요즘 TIL 에 소홀했다. 어제 오리가 블로그 쓴다기에 오리 블로그를 처음 봤는데, 담백한 어투가 참 오리답다고 생각했다. 나도 다시 TIL 을 열심히 써보려 한다. 땡큐 오리. 카카오 코테 2.9솔 했다. 시간이 좀 더 있었다면 한 문제 더 풀 수 있었을텐데 아쉽다. 역시 알고리즘은 연습이 필요하다. 코테도 좋은 경험이었다. 떨어지더라도…

https://vero.wiki/2023-11-25/

**Comment by carsago** [comment]
Thank you Vero
